packages/next-mdx/package.json

```diff
@@ -5,10 +5,10 @@
   "license": "MIT",
   "repository": "zeit/next-plugins",
   "dependencies": {
-    "@mdx-js/loader": "^0.15.0"
+    "@mdx-js/loader": "^0.16.0"
```

Comment:
I think we should make these all peerDependencies so that we don't have to update next-mdx when new versions come out. 

Then we can do something like eslint-config-airbnb does and recommend:

```
npx install-peerdeps @zeit/next-mdx
```

Reply:
What versioning should be used? Because `^0.x := >=0.0.0 <1.0.0`, I think I would recommend using `^0.x` until either of them reach `1.0.0`, since `mdx` seems to follow semver, but I'm not an expert. 😂

```json
{
  "peerDependencies": {
    "@mdx-js/loader": "^0.x",
    "@mdx-js/mdx": "^0.x"
  }
}
```

I can change anything that is needed, and also add the one-liner you mention to the `README.md`, once we agree on how to move forward!

Reply:
I'd just do `">=0.0.1"` there's no way it'll break the loader implementation for webpack 😄

Reply:
Updated this PR in line with what we discussed, plus updated the readme.